test/test_task_on_kart.py

```diff
@@ -2,8 +2,8 @@
 import pathlib
 import unittest
 from datetime import datetime
-from typing import Any, Dict
-from unittest.mock import MagicMock, patch
+from typing import Any, Dict, List, cast
+from unittest.mock import Mock, patch
```

Comment:
`isinstance(MagicMock, Iterable)` returns true. It is difficult to test, so I use `Mock` instead of `MagicMock`.